### DISCUSSION TITLE
Can't make a Vercel project of a project I have collaborate access to.

### DISCUSSION THREAD

**Discussion by iHildy** [vercel/community#2403] [Help]
### Summary

I have collaborator access to a private GitHub repository. In the new projects page of Vercel none of my repositories that I have collaborator access show up. All of the ones that I own show up correctly though. My Vercel account was Oauth'd with my GitHub account so I don't *think* that should be the issue, maybe there's a GitHub or Vercel outage? [This](https://vercel.com/docs/concepts/deployments/git/vercel-for-github#missing-git-repository) help article describes that I should have access to create projects with collaborator access: https://vercel.com/docs/concepts/deployments/git/vercel-for-github#missing-git-repository

**Comment by amyegan** [comment]
Hi @iHildy! When creating a new project, do you have additional options in the dropdown under "Import Git Repository"?  

![image](https://github.com/vercel/community/assets/8247105/d49fffcd-440a-4e5a-a6a3-b4a5b2c14b57)

**Comment by iHildy** [comment]
Hey @amyegan I do, it's my organization profile but that has nothing to do with my issue or the repositories I'm trying to access. That's it though apart from the standard options.

**Reply by simkessy** [reply]
seeing the same thing

**Reply by amyegan** [reply]
If you're able to select the correct GitHub account or organization, I would expect the repo with collaborator access to appear after you type its name in the search box. Can you share a screenshot or video of where the problem is happening and a description of what you expect to see?

**Comment by dr-debugger** [comment]
has the issue resolved yet?

**Comment by simonandywilson** [comment]
I'm still having this issue when trying to access a collaborative repository.
